stories/helperComponents/MockSettings.tsx

```diff
@@ -151,7 +145,7 @@ export class MockSettings extends React.PureComponent<
       const tpaIdx = parseInt(item.defaultColor.split('-')[1], 10) - 1;
       const { row, col } = this.getPaletteIndicesFromTPAIdx(tpaIdx);
       obj[item.wixParam] = {
-        value: COLOR_PALETTE[col][row],
+        value: palettes[0][col][row],
```

Comment:
This part was just wrong.
It caused a bug where the initial colors were wrong, because the `COLOR_PALETTE` was the old paletee, before our refactor.